src/mcp_mikrotik/scope/wireless.py

```
@@ -0,0 +1,640 @@
+    if time:
+        cmd += f" time={time}"
+
+    if comment:
+        cmd += f' comment="{comment}"'
+
+    result = execute_mikrotik_command(cmd)
+
+    if "failure:" in result.lower() or "error" in result.lower():
+        return f"Failed to create wireless access list entry: {result}"
+
+    return f"Wireless access list entry created successfully."
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)